extract_unique_sensors_01.py

```diff
@@ -52,14 +52,14 @@ def read_sensors(file_name, url):
     sensors_data = []
     if file_name and Path(file_name).exists():
         file_path = Path(file_name)
-        logging.debug("Accesing sensor data from the file_name: %r.", file_name)
+        logging.debug("Accessing sensor data from the file_name: %r.", file_name)
         with open(file_path) as lines:
             sensors_data = lines.readlines()
     elif url:
         try:
             sensors_url = urlopen(url)
             sensors_data = sensors_url.readlines()
-            logging.debug("Accessing sensor data from the url:  %r.", url")
+            logging.debug("Accessing sensor data from the url:  %r.", url)
```

Comment:
Fixed a syntax error, I was hoping that `flake8` or your tests would pick it up.